cavityFaultDisplay.py

```diff
@@ -74,6 +83,24 @@ def __init__(self, parent=None, args=None, macros=None):
             horizontalLayout.addWidget(shortDescriptionLabel)
             horizontalLayout.addWidget(statusLabel)
 
+            if fault.button_level == "GUI":
+                button = PyDMRelatedDisplayButton(filename=fault.button_command)
+
+            elif fault.button_level == "EDM":
+                button = PyDMEDMDisplayButton()
+                button.filenames = [fault.button_command]
+                button.macros = fault.macros + ',' + fault.button_macro
+
+            elif fault.button_level == "PyDM":
+                button = PyDMRelatedDisplayButton()
+                button.filenames = [fault.button_command]
+
+            else:
```

Comment:
It looks like GUI, PyDM, and other are all the same, so I'd probably just do if EDM, else

Reply:
Okay, updated!